src/Core/Form/IdentifiableObject/Handler/FormHandler.php

```diff
@@ -144,6 +144,7 @@ private function handleFormUpdate(FormInterface $form, $id)
 
         $this->hookDispatcher->dispatchWithParameters('actionAfterUpdate' . Container::camelize($form->getName()) . 'FormHandler', [
             'id' => $id,
+            'form_data' => $data,
```

Comment:
If this data is used by many modules, should it be passed by reference in order to get changes between modules? :thinking:

Reply:
The usecase @PierreRambaud think of is:
- module 1 that is hooked here modifies `form_data`
- module 2 that is hooked here too wants to modify `form_data` but it gets the initial `form_data`, without the modifications done by module 1

Reply:
Requested change applied in https://github.com/PrestaShop/PrestaShop/pull/14008/commits/3d4336b5338b87f6c5888ea1cec48b6baac2786d